packages/package_info_plus/package_info_plus_macos/pubspec.yaml

```diff
@@ -16,5 +16,6 @@ environment:
   flutter: ">=1.20.0"
 
 dependencies:
+  package_info_plus_platform_interface: ^2.0.0
```

Comment:
Is there still a reason to add this dependency here?

Reply:
No, it doesn't seem so. I'll remove it

Reply:
```suggestion
```